api/services/account_service.py

```diff
@@ -84,10 +113,13 @@ def load_user(user_id: str) -> None | Account:
         return account
 
     @staticmethod
-    def get_account_jwt_token(account, *, exp: timedelta = timedelta(days=30)):
+    def get_account_jwt_token(account: Account) -> str:
         payload = {
             "user_id": account.id,
-            "exp": datetime.now(timezone.utc).replace(tzinfo=None) + exp,
+            "exp": int(
+                datetime.now(timezone.utc).timestamp()
+                + timedelta(minutes=dify_config.ACCESS_TOKEN_EXPIRE_MINUTES).total_seconds()
```

Comment:
Considering the expiration time is used in seconds, it's suggested to change the time unit of this config from minutes to seconds for better consistency and flexibility.

Reply:
Thank you for your suggestion! For this scenario, using minutes is sufficient and more readable, as there won't be any access tokens with a validity period of less than one minute.

Reply:
SGTM.